src/components/draggable/draggable.spec.tsx

```diff
@@ -75,38 +75,19 @@ describe("Draggable", () => {
 
   it("should return an array with id's", () => {
     wrapper.setProps({ getOrder });
-    wrapper.find("DropTarget").at(0).props().getOrder(2);
+    (wrapper.find("DropTarget").at(0).props() as any).getOrder(2);
```

Comment:
I think this is due to the way we are finding the `"DropTarget"` as it seems to return `HTMLAttributes` rather than the component type with props etc. I think we can move `DropTarget` into and `internal` directory and import it in the container files and then remove `any`